### PR TITLE
Added styles to Sign Up and Sign In component

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: .firebase/hosting.ZGlzdFxidWRnZXQtdHJhY2tlcg.cache [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
+styles-es5.js.map,1582406841560,9d1fea7dddc0be07abaf85fd8b2b4e17a2a8f94601440795c05340357cb716d8
+authentication-authentication-module-es5.js,1582407800630,8aa5659ab2f83d31cb1846f05c70ed55aa48f3e10f597754a18c14ad5daf1e7f
+polyfills-es2015.js,1582406840469,bd34044b48a1178b6244f3d396bc5c4c3d31b78d5c6f1b34c3056256b71331e9
+polyfills-es2015.js.map,1582406840458,df35114a4297193a179ba94c31af00be4b502c3a770c0297c3f8405a7031ec09
+polyfills-es5.js,1582406843779,e9b80f8eeabd1d4a0310240595c8cabb0a97f3d33feabce30a4abb0d57390e3f
+polyfills-es5.js.map,1582406843762,f069e1d37fc78026d6eb246b0c648b9b7929b579d395de673a8cf963510b4879
+vendor-es2015.js,1582406854838,fb678162b1fe85ee392dc26f3f112cf2161ef39e570704fc96107c1303f2d98c
+vendor-es2015.js.map,1582406854768,1455a2191b1d7c6605f22d8d42e0474f21f25b4d214570b416c2664cd0e5c846
+vendor-es5.js,1582406854653,49320bd680c1e9c635b252f54afe7bf6e10bb7ca5d1299f8904ff9cf101383ab
+vendor-es5.js.map,1582406854560,0460294e763d6b742a8b63590c37a75049de101e5d5f7f73f020fb065640d370
```
Comment (Rementh) [inline .firebase/hosting.ZGlzdFxidWRnZXQtdHJhY2tlcg.cache]: Do no include firebase cache in git repository. File is autogenerated and should not be committed. Please add whole .firebase directory into .gitignore.

File: src/app/authentication/sign-in/sign-in.component.html [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 
-<!--<button (click)="signInWithGoogle()">Sign In with Google</button>-->
-<!--<button routerLink="../signup">Don't have an account?</button>-->
-<!--<button routerLink="/dashboard">Kliknij sobie</button>-->
-<!--<button (click)="signOut()">Log out</button>-->
+  <p class="right">Forgot password?</p>
+
+  <div class="btn-container">
+    <button standardBtn type="submit">Sign In!</button>
+    <p>Don't you have an account? <a [routerLink]="['/auth/signup']">Sign Up!</a></p>
```
Comment (Rementh) [inline src/app/authentication/sign-in/sign-in.component.html]: There is no need to use data binding with static path on routerLink. Try to use binding only when you have something to bind, like url variable. For static string paths, just use routerLink='/auth/signup'.

File: src/app/authentication/sign-in/sign-in.component.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ import {NgForm} from '@angular/forms';
 @Component({
   selector: 'app-sign-in',
   templateUrl: './sign-in.component.html',
-  styleUrls: ['./sign-in.component.scss']
+  styleUrls: ['./sign-in.component.scss', '../authentication.component.scss']
```
Comment (Rementh) [inline src/app/authentication/sign-in/sign-in.component.ts]: Why are there styles from another component? That should be encapsulated for one scss file per component.

File: src/app/authentication/sign-up/sign-up.component.ts [hunk 4]
```diff
@@ -5,7 +5,7 @@ import {AuthenticationService} from '../../shared/services/authentication.servic
 @Component({
   selector: 'app-sign-up',
   templateUrl: './sign-up.component.html',
-  styleUrls: ['./sign-up.component.scss']
+  styleUrls: ['./sign-up.component.scss', '../authentication.component.scss']
```
Comment (Rementh) [inline src/app/authentication/sign-up/sign-up.component.ts]: The same as before.

File: src/app/shared/directives/buttons/standard-btn.directive.ts [hunk 5]
```diff
@@ -13,9 +13,9 @@ export class StandardBtnDirective implements OnInit {
     textDecoration: 'none',
     outline: 'none',
     width: '300px',
-    height: '50px',
+    padding: '0.8rem 0',
     borderRadius: '30px',
-    margin: '1.8rem 0 1.8rem 0',
+    margin: '1rem 0 1rem 0',
```
Comment (Rementh) [inline src/app/shared/directives/buttons/standard-btn.directive.ts]: Can be shorten to '1rem 0'.

File: src/styles.scss [hunk 6]
```diff
@@ -3,11 +3,17 @@
   font-family: "Montserrat", sans-serif;
 }
 
-h1, h2, h3, h4, h5, h6, p {
+body {
   margin: 0;
+  height: 100vh
```
Comment (Rementh) [inline src/styles.scss]: Missing semicolon? :D

**Comment by ewelina-slepko** [comment]
Done! Thanks :D

**Pull request merged**